enterprise/api/v1/views.py

```diff
@@ -315,7 +313,6 @@ def enroll_learners_in_courses(self, request, pk):
             }
             if len(pending_users | existing_users) > 0:
                 LOGGER.info("Successfully bulk enrolled learners: {}".format(pending_users | existing_users))
-                track_enrollment('customer-admin-enrollment', request.user.id, course_run)
```

Comment:
removing this, since a customer-admin-enrollment tracking is already sent from inside the function `customer_admin_enroll_user_in_course` @taliaedX lmk if any flags

Reply:
undone! we want this tracking too, fixed test as well